content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,308 @@
+    unlikely while consuming less resources. A larger `handSize` also increases the
+    effective concurrency share available to a single flow in an overload situation.
+    {{< /note >}}
+
+### FlowSchema
+A FlowSchema matches some inbound requests and assigns them to a
+PriorityLevelConfiguration. Every inbound request is matched against every
+FlowSchema in turn, starting with those with highest `MatchingPrecedence` and
+working down. 
+
+{{< caution >}}
+Only the first matching FlowSchema for a given request matters.
+If multiple FlowSchemas match a single inbound request, it will be assigned
+based on the one with the highest `MatchingPrecedence`.
+{{< /caution >}}
+
+A FlowSchema matches a given request if at least one of its `rules`
+matches. A rule matches if at least one of its `subjects` *and* at least
+one of its `resourceRules` or `nonResourceRules` (depending on whether the
+incoming request is for a resource or non-resource URL) matches. 
+
+For the `name` field in subjects, and the `verbs`, `apiGroups`, `resources`,
+`namespaces`, and `nonResourceURLs` fields of resource and non-resource rules,
+the wildcard `*` may be specified to match all of the appropriate object,
+effectively removing it from consideration.
+
+A FlowSchema's `flowDistinguisherMethod` determines how requests matching that
+schema will be separated into flows for the fair queuing algorithm. It may be
+either `ByUser`, in which case one requesting user will not be able to starve
+other users of capacity, or `ByNamespace`, in which case requests for resources
+in one namespace will not be able to starve requests for resources in other
+namespaces of capacity. The correct choice for a given FlowSchema depends on the
+resource and your particular environment.
+
+## Diagnostics
+Every HTTP response from an API server with the priority and fairness feature
+enabled will have two new headers, `X-Kubernetes-PF-FlowSchemaUID` and
+`X-Kubernetes-PF-PriorityLevelUID`, noting the flow schema that matched the request
+and the priority level to which it was assigned, respectively. The API objects'
+names are not included in these headers in case the requesting user does not
+have permission to view them, so when debugging you can use a command like 
+
+```shell
+kubectl get flowschemas -o custom-columns="uid:{metadata.uid},name:{metadata.name}"
+kubectl get prioritylevelconfigurations -o custom-columns="uid:{metadata.uid},name:{metadata.name}"
+```
+
+to get a mapping of UIDs to names for both FlowSchemas and
+PriorityLevelConfigurations.
+
+## Observability
+When you enable the API Priority and Fairness feature, the kube-apiserver
+exports additional metrics. Monitoring these can help you determine whether your
+configuration is inappropriately throttling important traffic, or find
+poorly-behaved workloads that may be harming system health.
+
+* `apiserver_flowcontrol_rejected_requests` counts requests that were rejected,
```

Comment:
https://github.com/kubernetes/kubernetes/pull/88311#discussion_r382892843

the instrumentation guys are suggesting applying a "xxx_total" suffix for this metrics

Reply:
Deferring the change here to someone making that change in code